cinq_auditor_required_tags/providers.py

```diff
@@ -191,13 +191,19 @@ def delete_s3_bucket(client, resource):
                     resource.resource_id,
                     resource.account
                 ))
-
-            if not 'cinqDenyObjectUploads' in bucket.Policy().policy:
+            try:
+                current_bucket_policy = bucket.Policy().policy
+            except botocore.exceptions.ClientError:
+                if error.response['Error']['Code'] == 'NoSuchBucketPolicy':
+                    current_bucket_policy = None
```

Comment:
you'll need to make current_bucket_policy a string value if you want to iterate with the if.